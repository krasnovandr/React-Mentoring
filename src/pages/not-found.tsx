import * as  React from "react";
import Grid from "@material-ui/core/Grid";

export function NotFound() {
    return (
        <Grid container spacing={24} direction="column" alignItems="center" justify="center">
            <Grid item xs={12} >
                <h1>Page which you are looking for doensn't exist</h1>
            </Grid>
        </Grid>
    )
}
