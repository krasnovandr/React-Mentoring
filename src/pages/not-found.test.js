import React from 'react'
import { render, cleanup } from 'react-testing-library'
import renderer from 'react-test-renderer';
import { NotFound } from './not-found';
afterEach(cleanup)
describe('NotFound  Component', () => {
    it('should render error component', () => {
        const component = renderer.create(
            <NotFound />
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

    })
})