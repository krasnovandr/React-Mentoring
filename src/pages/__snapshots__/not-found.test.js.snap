// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound  Component should render error component 1`] = `
<div
  className="MuiGrid-container-1 MuiGrid-spacing-xs-24-25 MuiGrid-direction-xs-column-4 MuiGrid-align-items-xs-center-9 MuiGrid-justify-xs-center-18"
>
  <div
    className="MuiGrid-item-2 MuiGrid-grid-xs-12-41"
  >
    <h1>
      Page which you are looking for doensn't exist
    </h1>
  </div>
</div>
`;
